billy/app.py

```diff
@@ -70,6 +79,18 @@ class Meta:
 bulkInvoiceSchema = BulkInvoiceSchema()
 bulkInvoicesSchema = BulkInvoiceSchema(many=True)
 
+@app.route('/upgrade-db', methods=['POST'])
```

Comment:
we are there public routes for this?

Reply:
Removed routes, added "before_first_request" hook for database upgrades